File: app/src/main/java/org/metabrainz/mobile/presentation/features/suggestion/SuggestionProvider.java
package org.metabrainz.mobile.presentation.features.suggestion;

import android.content.SearchRecentSuggestionsProvider;

public class SuggestionProvider extends SearchRecentSuggestionsProvider {

    public final static String AUTHORITY = "org.metabrainz.mobile.SearchSuggestionProvider";
    public final static int MODE = DATABASE_MODE_QUERIES;

    public SuggestionProvider() {
        setupSuggestions(AUTHORITY, MODE);
    }

}

File: app/src/main/java/org/metabrainz/mobile/presentation/features/suggestion/SuggestionProvider.kt
package org.metabrainz.mobile.presentation.features.suggestion

import android.content.SearchRecentSuggestionsProvider

class SuggestionProvider : SearchRecentSuggestionsProvider() {
    companion object {
        const val AUTHORITY = "org.metabrainz.mobile.SearchSuggestionProvider"
        const val MODE = DATABASE_MODE_QUERIES
    }

    init {
        setupSuggestions(AUTHORITY, MODE)
    }
}